build_runner/lib/src/entrypoint/options.dart

```diff
@@ -54,11 +52,15 @@ class _SharedOptions {
 
   final bool enableLowResourcesMode;
 
+  /// Path to the merged output directory, if applicable.
```

Comment:
[nit] can we be more explicit about what 'applicable' means?

`/// Path to the merged output directory, or null if no directory should be created.`